ports/libjson/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# QNX CMakeLists.txt
+# Made for cross-compiling libjson with QNX
+
+#--------------------------------------------
+# Verify correct version
+cmake_minimum_required ( VERSION 3.5 )
+
+#--------------------------------------------
+# Set up environment variables
+set ( MAJOR_VERSION 1 )
+set ( MINOR_VERSION 0 )
+set ( MICRO_VERSION 0 )
+
+#--------------------------------------------
+# Library
+add_library ( json SHARED
+    json.c
+)
+set_target_properties ( json PROPERTIES
+    VERSION             ${MAJOR_VERSION}.${MINOR_VERSION}.${MICRO_VERSION}
+    SOVERSION           ${MAJOR_VERSION}
+)
+
+#-------------------------------------------
+# Test Executable
+add_executable ( jsonlint
+    jsonlint.c
+    json.c
+)
+
+#-------------------------------------------
+# Installation
+install (
+    TARGETS             json
+    DESTINATION         lib
+)
+install (
+    TARGETS             jsonlint
+    DESTINATION         local/libjson_tests
```

Comment:
We usually configure the install location with,
```
PREFIX ?= /usr/local
...
             -DCMAKE_INSTALL_LIBDIR=$(INSTALL_ROOT)/$(CPUVARDIR)/$(PREFIX)/lib \
             -DCMAKE_INSTALL_BINDIR=$(INSTALL_ROOT)/$(CPUVARDIR)/$(PREFIX)/bin \
             -DCMAKE_INSTALL_PREFIX="$(PREFIX)" \
             -DCMAKE_INSTALL_INCLUDEDIR=$(INSTALL_ROOT)/$(PREFIX)/include \
```
This way nothing gets installed to an "unclean" location in the SDP.

Reply:
Implemented with72d499407fb036683e1d73737af8a4fe493826fb !